src/condor_sysapi/phys_mem.cpp

```diff
@@ -24,21 +24,91 @@
 #include "sysapi_externs.h"
  
 #if defined(LINUX)
-#include <stdio.h>
+#include <cstdio>
+#include <algorithm>
+
+
+// Assuming a cgroup-style file with an integer
+// or a string that we don't care about in it,
+// return the 64 bit int or 0 if it is a string.
+int64_t file_size_contents(const char *filename) {
+	int64_t s = 0;
+	FILE *f = fopen(filename, "r");
```

Comment:
```suggestion
	FILE *f = safe_fopen_wrapper_follow(filename, "r");
```

Reply:
Not sure we need this, given that /cgroup filesystem is guaranteed not to have internal symlinks or other nonsense that safe_... protects against.

Reply:
If the filename is full path that's a constant in the same function, sure. When you're passing the filename in as a parameter, it's too easy for another caller to creep in unnoticed with a less-safe value.